packages/cli/src/sso/saml/types/requests.ts

```diff
@@ -5,4 +5,5 @@ export declare namespace SamlConfiguration {
 	type Read = AuthenticatedRequest<{}, {}, {}, {}>;
 	type Update = AuthenticatedRequest<{}, {}, SamlPreferences, {}>;
 	type Toggle = AuthenticatedRequest<{}, {}, { loginEnabled: boolean }, {}>;
+	type Test = AuthenticatedRequest<{}, {}, {}, {}>;
```

Comment:
Not sure if we need this type, since it's not really used anywhere.

Reply:
I could reuse one of the other authenticatedRequest ones, but wanted to keep things clearly separated. The type is not being used as such, but the middleware for the test endpoint requires and AuthenticatedRequest for it to check whether the user is an owner

Reply:
You can use `AuthenticatedRequest` directly in that case.